build_ngtf.py

```diff
@@ -237,9 +237,9 @@ def main():
     print("Target Arch: %s" % target_arch)
 
     # The cxx_abi flag is translated to _GLIBCXX_USE_CXX11_ABI
-    # For gcc 4.8 - this flag is set to 0 and newer ones, this is set to 1
+    # For gcc 7.3 - this flag is set to 0 and newer ones, this is set to 1
     # The specific value is determined from the TensorFlow build
-    # Normally the shipped TensorFlow is built with gcc 4.8 and thus this
+    # Normally the shipped TensorFlow is built with gcc 7.3 and thus this
     # flag is set to 0
     cxx_abi = "0"
```

Comment:
cxx_abi  set to 1?

Reply:
It's get's generated and overwritten correctly later on, and this is just the initialization.

Reply:
But sure, I'll change that.

Reply:
yes, saw that it is overwritten. Basically, the comment was not aligned
```
 # Normally the shipped TensorFlow is built with gcc 7.3 and thus this
    # flag is set to 0
```

Reply:
Good catch. I fixed it